### ISSUE TITLE
Document best practices for translation

### ISSUE THREAD

**Issue by samtstern** [firebase/FirebaseUI-Android#799]
Adding translations to FirebaseUI will add strings in a ton of languages, more than most developers want to support.

We should encourage developers to set their `resConfigs` value to only the locales they support throughout their application.  This saves in APK size and prevents a fragmented experience.

Ideally people would do:

```groovy
android {
  // ...
  defaultConfig {
    // ...
    resConfigs "auto"
  } 
}
```

https://developer.android.com/studio/build/shrink-code.html#unused-alt-resources
https://blog.egorand.me/a-curious-case-of-multiple-locales/

**Comment by samtstern** [comment]
See #771

**Comment by samtstern** [comment]
The fix for this issue has been released in `v2.1.0`.
